source/Octopus.Client/Model/LoginState.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Octopus.Client.Model
+{
+    public class LoginState
+    {
+        /// <summary>
+        /// Whether the client says it's using a secure connection. We need this because SSL offloading
+        /// can obscure this and the server cannot tell whether the client initiated the call using a secure
+        /// connection.
+        /// </summary>
+        public bool UsingSecureConnection { get; set; }
```

Comment:
Same here, this will be set automatically by the client, unless it is set explicitly.